core/vtk/ttkFTMTree/ttkFTMTreeUtils.h

```diff
@@ -16,28 +16,103 @@
 
 #include <vtkCellData.h>
 #include <vtkIntArray.h>
+#include <vtkMultiBlockDataSet.h>
 #include <vtkPointData.h>
 #include <vtkUnstructuredGrid.h>
 
 namespace ttk {
   namespace ftm {
 
+    void manageInconsistentArcsMultiParent(FTMTree_MT *tree) {
```

Comment:
that function does not seem to manipulate any vtk objects.
shouldn't be implemented instead in `core/base/ftmTree/`?

Reply:
That remark extends to all functions in the vtk layer which do not explicitly manipulate vtk objects.

Reply:
Indeed! Moved in the next commit.